sycl/include/CL/sycl/builtins.hpp

```diff
@@ -383,7 +390,7 @@ pown(T x, T2 y) __NOEXC {
 
 // genfloat powr (genfloat x, genfloat y)
 template <typename T>
-detail::enable_if_t<detail::is_genfloat<T>::value, T> powr(T x, T y) __NOEXC {
+detail::enable_if_t<__FAST_MATH_GENFLOAT(T), T> powr(T x, T y) __NOEXC {
```

Comment:
Based on the information from https://github.com/intel/llvm/issues/4595, I suppose it should be possible to improve `pow` function as well. Do you plan to make similar changes for other functions?

Reply:
Sure, it would be possible to do the same also for other built-ins.

The only thing that prevents this is the fact that just a small subset of built-ins is present in the SYCL specification as [``::native``](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_math_functions). Unfortunately, `pow` is not among them.

Reply:
I see. 
I suppose, right now we can expose target specific functions instead of `::native` counterparts.
To make this more portable in the future, we can add an extension enabling more `::native` functions.